package.json

```diff
@@ -15379,8 +15388,8 @@
 					"group": "1_gitlens_actions_1@1"
 				},
 				{
-					"command": "gitlens.createPatch",
-					"when": "!listMultiSelection && false && !gitlens:untrusted && !gitlens:hasVirtualFolders && webviewItem =~ /gitlens:(commit|stash)\\b/",
+					"command": "gitlens.graph.createPatch",
+					"when": "!listMultiSelection && webview == gitlens.views.graph && !gitlens:untrusted && !gitlens:hasVirtualFolders && webviewItem =~ /gitlens:(commit|stash)\\b/",
```

Comment:
We don't need the `webview == gitlens.views.graph` part, since we are already matching on our `webviewItem` context. And since the graph can be in both a view and an editor, this will actually break the editor version (since the id isn't the same)

Reply:
Also, this one should stay off for now (e.g. don't remove the `&& false`) because this command is for local patches, not cloud patches, which we don't yet support -- we started on them, but we haven't plumbed them back into the UI

Reply:
done